### PR TITLE
2.9 to 3.0 merge

### PR DESCRIPTION
**Pull request by jack-w-shaw**
Merges:
- https://github.com/juju/juju/pull/14830
- https://github.com/juju/juju/pull/14779

### CODE REVIEW

File: tests/suites/manual/task.sh [hunk 1]
```diff
@@ -4,6 +4,11 @@ test_manual() {
 		return
 	fi
 
+	if [[ ${BOOTSTRAP_PROVIDER:-} == "aws" ]]; then
```
Comment (hmlanigan) [inline tests/suites/manual/task.sh:7]: it feels like something else is broken if this is required. though a merge pr is not the place to fix it.

**Review by hmlanigan — COMMENTED** [review]
A card is okay for the comment.

Have these tests been run again 3.0 locally from the merge PR?

File: tests/suites/manual/task.sh (hunk 1 above)
Comment (jack-w-shaw) [reply tests/suites/manual/task.sh:7]: This is only required for pipelines that make use of the aws cli (which is fairly rare) and resolves the edge case where `aws` is configured to use a different region then the integration tests.

Perhaps it should be in main.sh, ut as you say, snot something to fix in a merge PR

**Comment by jack-w-shaw** [comment]
Yes, these tests pass on the merge PR

**Review by hpidcock — APPROVED** [review]
